### PR TITLE
#166816064 Implement social login 

### PR DESCRIPTION
**Pull request by halimahO**
#### What does this PR do?
Implement social login
#### Description of Task to be completed?
* Enable social login 
* Write tests
#### How should this be manually tested?
* Navigate to http://persephone-frontend-staging.herokuapp.com/auth/google
* Authorize sign in with google
* Replace google in the url with twitter or facebook to test for each of them
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#167913954](https://www.pivotaltracker.com/n/projects/2353116/stories/166816064)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by dharmykoya — APPROVED** [review]
Looks good to me. Well done Halima

**Review by devPinheiro — COMMENTED** [review]
Looks good to me. kindly resolve the conflicts on this PR.

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+API_URL=http://localhost:3000/api/v1
```
Comment (fxola) [inline .env]: Great work Halima. Is there any reason why you're using localhost for this variable

Comment (halimahO) [reply .env]: I used localhost to test the changes I made in the backend before it got merged and deployed and I forgot to remove it. Thanks for noting that.

**Pull request merged**